frontend/src/routes/(authenticated)/user/+page.ts

```diff
@@ -5,8 +5,17 @@ import type {
 
 } from '$lib/gql/types';
 import { getClient, graphql } from '$lib/gql';
-import { invalidate } from '$app/navigation';
-import {refreshJwt} from '$lib/user';
+import { goto, invalidate } from '$app/navigation';
+import { refreshJwt } from '$lib/user';
+import type { PageLoad } from './$types';
+import { browser } from '$app/environment';
+import type { EmailResult } from '$lib/email';
+
+export const load = (async ({ url }) => {
+  const emailResult = url.searchParams.get('emailResult') as EmailResult | null;
```

Comment:
can we validate that `emailResult` is a valid value? we could get a weird result somewhere if that gets messed up and I'd rather know early.

Reply:
Here it is: https://github.com/sillsdev/languageforge-lexbox/pull/172/commits/14dff8073f6cc810aa4c83a07a7d8d43873042fc